src/os/Transaction.h

```diff
@@ -241,9 +241,17 @@ class Transaction {
   ceph::buffer::list data_bl;
   ceph::buffer::list op_bl;
 
-  std::list<Context *> on_applied;
-  std::list<Context *> on_commit;
-  std::list<Context *> on_applied_sync;
+  std::list<std::unique_ptr<Context>> on_applied;
+  std::list<std::unique_ptr<Context>> on_commit;
+  std::list<std::unique_ptr<Context>> on_applied_sync;
+
+  static void _transform_contexts(
```

Comment:
Not sure what the '_' signifies.

Reply:
Just adhering a convention (`_` is already established in this file, e.g. `_get_next_op`).